DxDispatch/doc/Guide.md

```diff
@@ -39,33 +40,48 @@ This guide is organized as follows:
 
 # Running the Program
 
-The most basic usage is to simply pass in the path to a JSON model. For example, this will run on of the examples:
+The most basic usage is to simply pass in the path to a JSON model. For example, this will run one of the DML operator examples:
 
 ```
 > dxdispatch.exe .\models\dml_reduce.json
 
 Running on 'NVIDIA GeForce RTX 2070 SUPER'
+Dispatch 'sum': 1.1294 ms average
 Resource 'input': 1, 2, 3, 4, 5, 6, 7, 8, 9
 Resource 'output': 6, 15, 24
 ```
 
 There are more options available to you. If you run the executable with no arguments (or `--help`) it will display the available options:
 
 ```
-dxdispatch version 0.1.0
-Usage:
-  dxdispatch [OPTION...] <PATH_TO_MODEL_JSON>
+dxdispatch version 0.6.0
+  DirectML     : NuGet (Microsoft.AI.DirectML.1.8.2)
+  D3D12        : NuGet (Microsoft.Direct3D.D3D12.1.602.0)
+  DXCompiler   : Release (v1.6.2112)
+  PIX          : NuGet (WinPixEventRuntime.1.0.220124001)
+  ONNX Runtime : NuGet (Microsoft.ML.OnnxRuntime.DirectML.1.11.0)
 
-  -d, --debug                Enable D3D and DML debug layers
-  -b, --benchmark            Show benchmarking information
-  -i, --dispatch_repeat arg  [Benchmarking only] The number of times to      
-                             repeat each dispatch (default: 128)
-  -h, --help                 Print command-line usage help
-  -a, --adapter arg          Substring to match a desired DirectX adapter    
-                             (default: )
-  -s, --show_adapters        Show all available DirectX adapters
-  -q, --direct_queue         Use a direct queue/lists (default is compute
-                             queue/lists)
+Usage:
+  dxdispatch [OPTION...] <PATH_TO_MODEL>
+
+  -d, --debug                   Enable D3D and DML debug layers
+  -i, --dispatch_iterations arg
+                                The number of times to repeat each dispatch
+                                (default: 1)
+  -h, --help                    Print command-line usage help
+  -a, --adapter arg             Substring to match a desired DirectX adapter
+                                (default: )
+  -s, --show_adapters           Show all available DirectX adapters
+  -q, --direct_queue            Use a direct queue/lists (default is a
+                                compute queue/lists)
+      --xbox_allow_precompile   Disables automatically defining
+                                __XBOX_DISABLE_PRECOMPILE when compiling shaders for Xbox
+  -c, --pix_capture_type arg    Type of PIX captures to take: gpu, timing, or
+                                manual. (default: manual)
+  -f, --onnx_free_dim_override arg
+                                List of free dimension overrides by name
+                                (ONNX models only). Can be repeated. Example: -f
+                                foo:3 -f bar:5 (default: )
```

Comment:
>default:  [](http://example.com/codeflow?start=48&length=9)

default is no overrides?

(just looks funny having a colon with nothing after it)